packages/terra-table/src/TableCell.jsx

```diff
@@ -12,11 +12,14 @@ const propTypes = {
 const TableCell = ({
   content,
   ...customProps
-}) => (
-  <td {...customProps} data-class="cell">
-    {content}
-  </td>
-);
+}) => {
+  const dataClassNames = customProps['data-class'] ? `cell ${customProps['data-class']}` : 'cell';
```

Comment:
There is not a thread that discusses the decision to include data attributes, this was something @bjankord and I discussed offline during the CSS Module conversion. These are intended to be dom descriptions.

Before the CSS Module conversion, some elements had terra class names added to represent the structure of the component but did not apply any styling. In the conversion, the class names of un-styled elements were not locally scoped to the component because their names were not bound to the style sheet. We decided it was appropriate to keep these names, but change them to data attributes, to maintain the visual structure of our components.